frontend/src/components/form/base/__tests__/__snapshots__/EColorPicker.spec.js.snap

```diff
@@ -2,11 +2,16 @@
 
 exports[`An EColorPicker > looks like a color picker > pickerclosed 1`] = `
 <div data-app="true">
-     <div class="e-form-container"
-          name="test"
+     <div class="e-form-container e-picker-base"
+          label="test"
           vee-rules="[object Object]">
           <div class="v-menu">
-               <div class="e-form-container">
+               <div buttonarialabeli18nkey="components.form.base.eColorPicker.openPicker"
```

Comment:
why does this change this much?

Reply:
Fixed this with [69b4f47](https://github.com/ecamp/ecamp3/pull/4746/commits/69b4f47195199988c37424ac31714cb6a49fded9)